cmd/copyEnumeratorInit.go

```diff
@@ -409,6 +409,15 @@ func (cca *CookedCopyCmdArgs) InitModularFilters() []ObjectFilter {
 		}
 	}
 
+	switch cca.permanentDeleteOption {
+	case common.PermanentDeleteOption(0):
+		filters = append(filters, &softDeleteFilter{deleteSnapshots: true})
+	case common.PermanentDeleteOption(1):
```

Comment:
It should look like something similar to common.EPermanentDeleteOption.DeleteSnapshots()